linera-rpc/README.md

```diff
@@ -0,0 +1,14 @@
+<!-- cargo-rdme start -->
+
+This module provides network abstractions and the data schemas for remote procedure
+calls (RPCs) in the Linera protocol.
+
+<!-- cargo-rdme end -->
+
+## Contributing
+
+See the [CONTRIBUTING](../CONTRIBUTING.md) file for how to help out.
+
+## License
+
+This project is available under the terms of either the [Apache 2.0 license](../LICENSE).
```

Comment:
```suggestion
This project is available under the terms of the [Apache 2.0 license](../LICENSE).
```

Also in the other `README.md` files.
(Or should it be "Apache or MIT"?)

Reply:
I don't think we can make this change for now, strictly speaking.